src/components/forms/login-form.tsx

```diff
@@ -39,18 +40,29 @@ export const LoginForm = () => {
   };
 
   return (
-    <FormContainer>
-      <h1> Bem vindo(a) a Taqtile </h1>
-      <FormField label="Email" value={email} onChange={(value) => setEmail(value)} isButtonClicked={isButtonClicked} />
-      <FormField
-        label="Senha"
-        value={password}
-        onChange={(value) => setPassword(value)}
-        isButtonClicked={isButtonClicked}
-      />
-      {loading && <div>Carregando...</div>}
-      {!isAnyFieldEmpty && error && <ErrorMessage message={error.message} />}
-      <Button title="Entrar" onClick={handleFormSubmit} />
-    </FormContainer>
+    <>
+      {loading ? (
```

Comment:
Aqui no return vc não precisa de tanto aninhamento de componente e lógica uma maneira de simplificar seria:
```suggestion
  return loading ? (
    <Component>
  ) : (
    <Form>
  )
```